src/compile_cache.cc

```diff
@@ -77,10 +77,25 @@ v8::ScriptCompiler::CachedData* CompileCacheEntry::CopyCache() const {
 // See comments in CompileCacheHandler::Persist().
 constexpr uint32_t kCacheMagicNumber = 0x8adfdbb2;
 
+const char* CompileCacheEntry::type_name() const {
+  switch (type) {
+    case CachedCodeType::kCommonJS:
+      return "CommonJS";
+    case CachedCodeType::kESM:
+      return "ESM";
+    case CachedCodeType::kStrippedTypeScript:
+      return "StrippedTypeScript";
+    case CachedCodeType::kTransformedTypeScript:
+      return "TransformedTypeScript";
```

Comment:
Does this flavor exist? I thought if we were transforming, we were adding source maps. Is there a way to transform without getting source maps?

Reply:
Im dont think its possible unless there is a way to explicitly disable sourcemaps.

Reply:
There is. See test/parallel/test-compile-cache-typescript-transform.js in this PR.